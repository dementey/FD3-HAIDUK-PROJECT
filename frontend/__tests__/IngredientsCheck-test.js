"use strict";

import React from 'react';
import renderer from 'react-test-renderer';

import IngredientsCheck from '../src/Components/LandingPage/IngredientsCheck/IngredientsCheck';

test('работа IngredientsCheck', () => {

  const component = renderer.create(
    <IngredientsCheck />
  );

  let componentTree=component.toJSON();
  expect(componentTree).toMatchSnapshot();

    
});
