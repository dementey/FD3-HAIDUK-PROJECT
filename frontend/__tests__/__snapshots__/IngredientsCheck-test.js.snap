// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`работа IngredientsCheck 1`] = `
<div
  className="home-box container"
>
  <form>
    <h2
      className="MuiTypography-root-1 MuiTypography-title-7 MuiTypography-gutterBottom-18"
    >
      Инструмент проверки ингредиентов на вредность
    </h2>
    <textarea
      autoFocus={true}
      className="form-control"
      onChange={[Function]}
      onKeyPress={[Function]}
      placeholder="Введите или вставьте ингридиенты продукта"
      rows="5"
    />
    <input
      className="btn btn-primary btn-lg btn-block"
      onClick={[Function]}
      type="button"
      value="Проверить"
    />
  </form>
</div>
`;
